1주차 step2 dnwhd0112

dnwhd0112 opened the pull request:
step2 과제입니다.

dnwhd0112 committed:
1주차 step2 dnwhd0112

Kioding commented on CodeStarterCamp_Week1/main.swift:
```diff
@@ -8,5 +8,22 @@
 
 import Foundation
 
-print("Hello, World!")
```
불필요한 코드의 삭제 좋습니다👍

Kioding commented on CodeStarterCamp_Week1/main.swift:
```diff
@@ -8,5 +8,22 @@
+printBar(count:4)
+
+
+func printIce(count : Int) -> Void{
```
- `Void{` <- 붙어있는 이유가 따로 있을까요?
- 반환값이 없는 표현식이라면 다른 방법으로도 표현할 수 있을까요🤔

dnwhd0112 replied:
지금 생각해보니 없는게 더 괜찮아 보이네요 처음이라 연습겸 리턴타입을 명확하게 써주는게 더 좋을것이라 생각했습니다.

Kioding commented on CodeStarterCamp_Week1/main.swift:
```diff
@@ -8,5 +8,22 @@
+
+
+func printIce(count : Int) -> Void{
+    for i in 1...count{
```
- 값을 직접 넣지 않고 `count`라는 변수를 넣어주셨네요! 좋은 포인트입니다🥳
- 그런데 `count`는 무슨 뜻인지 알기 어려운 것 같은데 어떻게 네이밍을 정하면 좋을까요?

dnwhd0112 replied:
length 가 적당할꺼 같네요. 아님 iceLenth? 가 더 괜찮을꺼같습니다.

Kioding replied:
그 네이밍 좋은데요! `iceCreamLength` or `lengthOfIceCream` 으로 하셔도 무방하겠네요 :)

Kioding commented on CodeStarterCamp_Week1/main.swift:
```diff
@@ -8,5 +8,22 @@
+}
+
+func printBar(count : Int) -> Void{
+    for i in 1...count{
```
- `i`는 무슨 뜻으로 쓰인 걸까요? 
- 실제 코드에서는 사용되고 있지는 않네요😃

dnwhd0112 replied:
반복문을 쓸때 꼭 필요한줄 알았습니다... ㅎ 변수가 필요없이 반복문을 쓸수 있는지 알아볼게요.

dnwhd0112 committed:
뺴뺴로 만들기

dnwhd0112 committed:
뺴뺴로 추가

dnwhd0112 committed:
Delete Make_paparow directory

dnwhd0112 committed:
2주차 commit templete 참조

dnwhd0112 committed:
아이스크림 막대기 과제입니다.
순서도는 추후 변경하여 commit 할 예정입니다.

순서도 URL 링크입니다.

https://viewer.diagrams.net/?tags=%7B%7D&highlight=0000ff&edit=_blank&layers=1&nav=1&title=1%E1%84%8C%E1%85%AE%E1%84%8E%E1%85%A1step2.drawio#R7VpLe6M2FP01WiYfb8QSsDNdTKfpl3Q6WcogAw22qJBj019fCYQxDzvMENuTNqtIR0%2Fu0bm6ujHQ%2FdXuE0VZ%2FCsJcQo0JdwBfQY0TVMsi%2F8RSFEhqmJLJKJJKLEGeEj%2BwXVHiW6SEOetjoyQlCVZGwzIeo0D1sIQpWTb7rYkaXvVDEW4BzwEKO2jfyYhiysUanaD%2F4KTKK5XVi2nalmhurP8kjxGIdkeQPoc6D4lhFWl1c7HqbBebZdq3N2R1v3GKF6zMQPiCC8Wf%2F%2F%2BLYgfP2dJ9Mdu%2BZTeaNUsLyjdyA8Gcx84JoBGWZgBryxAF7i84AHXBJ7PR5StcwDNElQEzgueAhxPDoRK2W0GHBc4d%2BUkEDiw7G8D6IsmqDazeRZw%2FIPZZmIbFQI5aNxKM7Ki5oaSzTrE4vMUoHvbOGH4IUOBaN3y48ixmK1SXlN5sW8uacEXTBneHUDSfJ8wWWFGC95FtuqmNJc8zAaU1G6bk6HWdMcHp8KSGJKHMdpP3fDFC5Ky76DP7tPXNRIO%2BXmWVUJZTCKyRum8Qb22GZs%2BnwnJpPH%2BwowVUpxow8hI0zJEI8xO7F%2F6ArHHkwRQnCKWvLQ1OWRNOfSeJHwre%2BIMBbaIM40OITnZ0ADLUR1O9tsYRdPj193i6VnbPAbBF%2FhcmF%2Bze3Sjwp%2BIp46xB4mr7PE6cYcED3%2B5PZHQSfKwBrxb5aQ%2BfJnRloSlXdCXDR6Vq0iCG5AW38T4W7OuPsnpysps16oVsvZjUhqWyEgp6edxjTXH9Z2mdPit9nU%2B1%2Fh9AUgpM7eSlnVUb5qVckt6C8pLkSiVwvOFjOcu8KDQdu%2BwpSmPKfFY8fWUNkD%2BiUCibXR9pPbgubRn9lj48ttV5ai2xNho8%2Fxy1MfebCcZvlFuoVZf%2BxMlaqnt02KOU6hLKSoOumWiQ35iGau9TH0q70b2t1vdeaHaQGdwvRuyXOb4LB6lH2s9zR%2FeT7g16VBeKto6tckfe0sORkzuibDreOwmwMPwzZRD%2BLoi7nNFE%2Fek%2Bl0%2FCIvJarHJL3MHqGZLPqozcAk4l7wEajdz9VvgDYVjjxSO85o3t2zDmObNz%2B%2F51P575%2F27vrEMVnK6lu8bSMQce2n%2B7z3dPvq9nqcbSHu%2Bb5U4Y1UyNcE2ye7O62bPY5SVJxEVZcDKzZQ%2FYxbE0sx5ZUyLF8t2TOcvWMS1lUllkMunMj1gzjiyTNLUJymhHAzxEm3KpyHFfCK0KFdVSi4ZYgf1DNOEfzMWowIslrlvEC9fo%2ByRVI79zWRiK51Hod2XiTGgkm4u8%2B1UYvzXVKKOzbWoU5Mt0ww%2FkBT50MkxnQzcJpeVycC754OtfX7R%2BsnY6qe6PthqQjWnzdaevavRNRxYQ1WkE3qX0SVTuJ3%2Fn9hDQe0b5XB5tfmZQPVYbH5toc%2F%2FBQ%3D%3D

dnwhd0112 committed:
Merge branch 'yagom-academy:2_dnwhd0112' into 2_dnwhd0112

dnwhd0112 committed:
Merge branch '2_dnwhd0112' of https://github.com/dnwhd0112/swift-starter-week1 into 2_dnwhd0112

dnwhd0112 committed:
템플릿을 적용하여 수정완료하였습니다.

dnwhd0112 committed:
순서도를 수정하여 올렸습니다.

dnwhd0112 committed:
refactor : 1주차 step2

컨벤션 수정 및 의미없는 매개변수 count -> lengthOfIce,lengthOfBar 로 수정

dnwhd0112 committed:
Update README.md

dnwhd0112 committed:
Merge pull request #1 from dnwhd0112/patch-2

Update README.md
